cpp/src/ray/runtime/task/invocation_spec.h

```diff
@@ -0,0 +1,21 @@
+
+#pragma once
+
+#include <ray/core.h>
+#include <msgpack.hpp>
+
+namespace ray {
+namespace api {
+
+class InvocationSpec {
+ public:
+  TaskID taskId;
+  ActorID actorId;
+  int actorCounter;
+  int32_t funcOffset;
+  int32_t execFuncOffset;
```

Comment:
What's the difference between these two fields? Please document them.

Also, can we use a structure to wrap these fields? Just like `FunctionDescriptor` in Java.

Reply:
FunctionDescriptor been packaged in subsequent processes.